src/webgpu/api/validation/capability_checks/features/clip_distances.spec.ts

```diff
@@ -0,0 +1,115 @@
+import { range } from '../../../../../common/util/util.js';
+import { align } from '../../../../util/math.js';
+import { kMaximumLimitBaseParams, makeLimitTestGroup } from '../limits/limit_utils.js';
+
+function getPipelineDescriptorWithClipDistances(
+  device: GPUDevice,
+  testValue: number,
```

Comment:
I think it would be clearer to name this parameter `interStageShaderVariables`, i.e.
``` suggestion
  interStageShaderVariables: number,
```